content/millennium/dstu2/medications/immunization.md

```diff
@@ -49,15 +55,27 @@ Search for Immunizations that meet supplied query parameters:
 
 ### Parameters
 
- Name      | Required?          | Type          | Description
------------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
- `_id`     | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
- `patient` | This, or `_id`     | [`reference`] | The patient for the vaccination record. Example: `12345`
- `date`    | N                  | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'.
+ Name      | Required      | Type          | Description
+-----------|---------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`     | Conditionally | [`token`]     | The logical resource ID associated with the resource. This parameter is required if the `patient` parameter is not used. Example: `12345`
+ `patient` | Conditionally | [`reference`] | The patient for the vaccination record(s). This parameter is required if the `_id` parameter is not used. Example: `12345`
+ `date`    | No            | [`date`]      | Date range for the immunization administration(s). Example: `date=ge2020-01-01T08:00:00.000Z&date=le2020-01-31T17:00:00.000Z`
+
+_Implementation Notes_
 
-Notes:
+- When searching with the `_id` parameter:
+  - It can be provided with either a single reference, or a comma-separated list of references. Example `_id=1234` or `_id=1234,5678`
```

Comment:
I forget where we had this conversation. do we need to include this on every resource?